src/MoBi.R/RRegister.cs

```diff
@@ -39,6 +47,11 @@ private static void registerMinimalTypes(IContainer container)
          container.Register<IMoBiConfiguration, IApplicationConfiguration, MoBiConfiguration>(LifeStyle.Singleton);
          container.Register<IMoBiXmlSerializerRepository, MoBiXmlSerializerRepository>(LifeStyle.Singleton);
          container.Register<IHistoryManagerFactory, HistoryManagerFactory>(LifeStyle.Singleton);
+         container.Register<IDiagramManagerFactory, DiagramManagerFactory>(LifeStyle.Singleton);
+         container.Register<IDimensionValidator, DimensionValidator>(LifeStyle.Singleton);
+         container.Register<IPresentationUserSettings, IUserSettings, ICoreUserSettings, BatchUserSettings>(LifeStyle.Transient);
+         container.Register<Services.ISimulationFactory, Services.SimulationFactory>(LifeStyle.Singleton);
```

Comment:
That one should be included by registration convention and **should** not need to be explicitly registered. Try to remove.